python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -59,8 +59,14 @@
 }
 BIGQUERY_WRITE_DISPOSITION = {"replace": "WRITE_TRUNCATE", "append": "WRITE_APPEND"}
 
-NATIVE_AUTODETECT_SCHEMA_SUPPORTED_FILE_TYPES = {FileType.CSV, FileType.NDJSON}
-NATIVE_AUTODETECT_SCHEMA_SUPPORTED_FILE_LOCATIONS = {FileLocation.GS}
+NATIVE_AUTODETECT_SCHEMA_CONFIG: Mapping[
+    FileLocation, Mapping[str, list[FileType] | Callable]
+] = {
+    FileLocation.GS: {
+        "filetype": [FileType.CSV, FileType.NDJSON, FileType.PARQUET],
+        "method": lambda table, file: None,
```

Comment:
What does this do? I mean the `None`.

What would `supported_config["method"](table=table, file=file)` look like for gcs to bq?

Reply:
@feluelle We actually don't need to do anything in case of GCS to bigquery, the native load file takes care of this. So this is a dummy function we execute. To have a pluggable interface, down the line we can easily plugin any other location like
```
FileLocation.S3: {
 "filetype": [FileType.CSV, FileType.NDJSON, FileType.PARQUET],
 "method": someMethodThatHandlesForS3
}
def someMethodThatHandlesForS3():
 pass

```

Reply:
Worth adding that as comments in the code too